RetroBar/App.config

```diff
@@ -37,7 +37,10 @@
             <setting name="ShowMultiMon" serializeAs="String">
                 <value>False</value>
             </setting>
-            <setting name="UseSoftwareRendering" serializeAs="String">
+			<setting name="UseSoftwareRendering" serializeAs="String">
+				<value>False</value>
+			</setting>
```

Comment:
For the sake of consistency in code formatting, it is preferable to use spaces instead of tabs.

Reply:
That was fast!